packages/guides/src/Handlers/ParseDirectoryHandler.php

```diff
@@ -98,14 +101,23 @@ private function getDirectoryIndexFile(
             $hashedContentFromFilesystem[$itemFromFilesystem['basename']] = true;
         }
 
-        foreach (self::INDEX_FILE_NAMES as $indexName) {
+        if ($indexName !== null) {
```

Comment:
if the user configured an `indexName`  we can enforce the index to exist like this. So I would expect a throw in this if block.

Reply:
I don't really get what you are suggesting?